scripts/embed_docs.py

```diff
@@ -12,8 +12,13 @@
 
 async def start():
     """Create a DuckDB vector store and add the scanpy embeddings to it."""
-    vector_store = DuckDBVectorStore(uri=str(EMBEDDINGS_DIR / "scanpy.db"), llm=OpenAI(), embeddings=OpenAIEmbeddings(), chunk_size=512)
-    await vector_store.add_directory(f"scanpy_{VERSION}", pattern="*", metadata={"version": VERSION}, situate=True)
+    db_path = str(EMBEDDINGS_DIR / "scanpy.db")
+    vector_store = DuckDBVectorStore(uri=db_path, llm=OpenAI(), embeddings=HuggingFaceEmbeddings(), chunk_size=512)
+    print(db_path)
+    await vector_store.add_directory(
+        f"scanpy_{VERSION}/html", pattern="*.html", exclude_patterns=["index.html", "api/*.html"],
```

Comment:
I ignore api because it has a lot of links to methods (e.g. [pearson_residuals](link/to/it), but doesn't provide any new description about it, which adds a lot of duplication bloat and noise.

Instead the signal is in the human written docs.